packages/react-composites/src/composites/CallComposite/MediaGallery.tsx

```diff
@@ -55,10 +58,14 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         remoteVideoViewOption={remoteVideoViewOption}
         styles={VideoGalleryStyles}
         layout="floatingLocalVideo"
-        onRenderAvatar={props.onRenderAvatar}
+        onRenderAvatar={(userId, options) => (
```

Comment:
Soft recommendation: Consider memoization.

Reply:
Not sure how I can do memoization here.
What do you suggest?